strings/src/main/res/values/strings.xml

```diff
@@ -547,7 +551,11 @@
     </plurals>
     <string name="location_status_searching">Searching for location, please wait&#8230;</string>
     <string name="location_status_accuracy">Location accuracy: %.2f m</string>
+    <!-- Low-priority for localisation -->
```

Comment:
Is this helpful for translators @lognaturel? It's not something we've done before.

Reply:
I don’t think so. Active languages have many translators and an additional string is not a big deal!